srcpkgs/base-files/files/_vkpurge

```diff
@@ -0,0 +1,29 @@
+#compdef vkpurge
+
+_vkpurge() {
+	local line state
+
+	_arguments -C \
+		'1: :->subcmd' \
+		'*::kernel:->kernels'
+
+	case "$state" in
+	subcmd)
+		_values 'subcommand' \
+			'list[List removable kernel versions]' \
+			'rm[Remove removable kernel versions]'
+		;;
+	kernels)
+		case "$line[1]" in
+		list)
+			_arguments '*::version glob: '
+			;;
+		rm)
+			_values 'kernel' \
+				'all[All removable kernels]' \
+				$(vkpurge list all)
```

Comment:
From the docs:

> The argument; this indicates which command line or option argument we are completing.  For command arguments this generally takes the form `argument-n`, where  `n`  is the number of the argument, and for arguments to options the form `option-opt-n` where `n` is the number of the argument to option opt.  However, this is only the case if the command line is parsed with standard UNIX-style options and arguments, so many completions do not set this.

So I guess it's fine to leave the argument field empty in this case. But `_values` inserts `values` instead.
```suggestion
			local kernels=(
				'all:all removable kernels'
				$(vkpurge list all)
			)
			_describe -t kenrels 'kernel' kernels
```